app/(storyblok)/error.tsx

```diff
@@ -5,8 +5,8 @@ import { Masthead } from '@/components/Masthead';
 import { Container } from '@/components/Container';
 
 export default function Error({error, reset}: {
-  error: Error & { digest?: string }
-  reset: () => void
+  error: Error & { digest?: string };
+  reset: VoidFunction;
```

Comment:
I learned that this can replace `() => void`. A bit clearner

Reply:
Neat